### PR TITLE
Add `remark-lint-double-link` to list

### PR DESCRIPTION
**Pull request by Scrum**

### CODE REVIEW

**Comment by Scrum** [comment]
> Hi there! Nice work!
> 
> It is unclear to me from this sentence, and from the readme of the project, what the project does? So you can’t have two links anymore? Or not two of the same URLs? What if the titles are different? How about images?
> 
> Could you clarify that in this PR (and maybe also update the readme to add some more docs?)

Yes, of course, it checks only duplicate `url`, which helps to get rid of duplicates with the same link names and with different ones. The main policy is not to duplicate links to the same resources

I also updated the documentation, if you still have questions, I will be happy to answer and take measures to improve accessibility

**Comment by wooorm** [comment]
Right, thanks for clarifying! In that case, it seems to be close to [`remark-lint-no-duplicate-defined-urls`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-duplicate-defined-urls), but for a different construct.

Could you change the text in this PR, that explains the rule, to better match what it is doing?

**Comment by Scrum** [comment]
> Right, thanks for clarifying! In that case, it seems to be close to [`remark-lint-no-duplicate-defined-urls`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-duplicate-defined-urls), but for a different construct.
> 
> Could you change the text in this PR, that explains the rule, to better match what it is doing?

Of course, thanks for directing me for the better.

File: readme.md [hunk 1]
```diff
@@ -392,6 +392,8 @@ This list is ordered based on the name without prefix, so excluding
     — Ensure that the `href` of links has no trailing slash
 *   [`remark-lint-write-good`](https://github.com/zerok/remark-lint-write-good)
     — Wrapper for `write-good`
+*   [`remark-lint-double-link`](https://github.com/Scrum/remark-lint-double-link)
+    — This rule ensures that there will be no duplicate links URL in the list.
```
Comment (wooorm) [inline readme.md:396]:
```suggestion
    — Ensure the same URL is not linked multiple times
```

How about this? The other external rules start with `Ensure` and end without a `.`. I’d prefer them to be similar in that sense

Thanks!

Comment (Scrum) [reply readme.md:396]: Thanks, it looks better.  I will make changes.

Comment (wooorm) [reply readme.md:396]: (you can click on the button here on GH!)

Comment (Scrum) [reply readme.md]: Yes, I wrote from a mobile application and there is no file editing there or I didn’t find it (

**Comment by wooorm** [comment]
Thanks @Scrum, merged! ✨

**Pull request merged**